Makefile

```diff
@@ -389,10 +389,9 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
-protoVer=0.8
+protoVer=0.10
```

Comment:
I get an error with this. 0.9.0 works.

Reply:
This is because 0.10 is the future version.